test/Concurrency/sendable_conformance_checking.swift

```diff
@@ -0,0 +1,147 @@
+// RUN: %target-typecheck-verify-swift
+// REQUIRES: concurrency
+
+@available(SwiftStdlib 5.1, *)
+class NotSendable { // expected-note 8{{class 'NotSendable' does not conform to the 'Sendable' protocol}}
+}
+
+@available(SwiftStdlib 5.1, *)
+@available(*, unavailable)
+extension NotSendable: Sendable { }
+
+@available(SwiftStdlib 5.1, *)
+protocol IsolatedWithNotSendableRequirements: Actor {
+  func f() -> NotSendable
+  var prop: NotSendable { get }
+}
+
+// Okay, everything is isolated the same way
+@available(SwiftStdlib 5.1, *)
+actor A1: IsolatedWithNotSendableRequirements {
+  func f() -> NotSendable { NotSendable() }
+  var prop: NotSendable { NotSendable() }
+}
+
+// Okay, sendable checking occurs when calling through the protocol
+// and also inside the bodies.
+@available(SwiftStdlib 5.1, *)
+actor A2: IsolatedWithNotSendableRequirements {
+  nonisolated func f() -> NotSendable { NotSendable() }
+  nonisolated var prop: NotSendable { NotSendable() }
+}
```

Comment:
so in this case we get warnings on `A2().f()` ?

Reply:
No, we don't need warnings on `A2().f()` because it doesn't cross any actor boundaries.

Reply:
Ah because nonisolated, right... Thanks!